test/models/tag_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
we'll be using rspec and factorybot instead of the default rails specs and fixtures; feel free to turn of the autogeneration of fixtures and rails tests.

Reply:
I dont know how to do that but i'll look into that :)

Reply:
Followed these docs: https://stackoverflow.com/questions/8497668/disable-unit-testing-generators-in-rails